docs/image-redactor/index.md

```diff
@@ -107,6 +112,31 @@ Pre-requisites:
 Python script example can be found under:
 /presidio/e2e-tests/tests/test_image_redactor.py
 
+## Getting started (DICOM images)
+
+=== "Python"
+
+    Once the Presidio-image-redactor package is installed, run this simple script:
+    
+    ```python
+    import pydicom
+    from presidio_image_redactor import DicomImageRedactorEngine
+
+    # Set input and output paths
+    input_path = "path/to/your/dicom/file.dcm"
+    output_dir = "./output"
+
+    # Initialize the engine
+    engine = DicomImageRedactorEngine()
+
+    # Option 1: Redact from a loaded DICOM image
+    dicom_image = pydicom.dcmread(input_path)
+    redacted_dicom_image = engine.redact(dicom_image, fill="contrast")
+
+    # Option 2: Redact from DICOM file(s)
+    engine.redact_from_file(input_path, output_dir, padding_width=25, fill="contrast")
+    ```
+

```

Comment:
I think it's worth adding the documentation you added into the presidio-image-redator folder to this doc as well (data citation, notice for windows etc.)